salesagility/SuiteCRM
# SuiteP: Adding Support for No Module Filters

## Description



SuiteP: Adding Support for No Module Filters

## Motivation and Context

If a user doesn't use 'Module Menu Filters' the top menu on desktop doesn't display well.
## How To Test This


1. Admin
2. Profile
3. Advance
4. check Module Menu Filters
5. Click save

Pass condition: if you are in the desktop view the right nav items should stay inline with the items.

## Types of changes


### Final checklist
